code/game/objects/effects/anomalies.dm

```diff
@@ -196,6 +196,8 @@
 	var/canshock = FALSE
 	var/shockdamage = 20
 	var/explosive = TRUE
+	///Does the anomaly have reduced explosion size?
+	var/reduced_explosion = FALSE
 
 /obj/effect/anomaly/flux/Initialize(mapload, new_lifespan, drops_core = TRUE, _explosive = TRUE)
```

Comment:
So, explosive is an initial arg for flux anomalies, but reduced explosion is a var only set by SM. I think it would be wiser to just make this part of initial args or make a subtype that has a reduced explosion... but probably args. Maybe different modes like nonexplosive, small boom and big boom? just not this varset after init, you know?